osbs/build/user_params.py

```diff
@@ -129,8 +129,9 @@ def __init__(self, build_json_store=None):
             "arrangement_version",
             allow_none=True,
             default=REACTOR_CONFIG_ARRANGEMENT_VERSION)
+        self.build_from = BuildParam('build_from')
         self.build_image = BuildParam('build_image')
```

Comment:
make some comment here, that build_image can be also imagestream and it now depends on build_is_imagestream

Reply:
Fixed.